sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/interpreter/operator/date/BeamSqlDatetimeMinusExpression.java

```diff
@@ -65,9 +51,23 @@ public BeamSqlDatetimeMinusExpression(List<BeamSqlExpression> operands, SqlTypeN
     this.delegateExpression = createDelegateExpression(operands, outputType);
   }
 
+  @Override
```

Comment:
Looks like from this point on your changes to this file are just shifting functions and whitespace around with no functional change. These sorts of changes make reviews difficult and history less useful. You should probably undo this no-op change.

Reply:
Done